src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.taskobjects.Task;
+
+/**
+ * TaskList object that is just a container for the task list
+ */
+public class TaskList {
+    ArrayList<Task> taskList;
+
+    /**
+     * Default constructor for TaskList
+     *
+     * Takes in no argument and creates an empty task list
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Alternative constructor for TaskList
+     *
+     * Takes in an {@code ArrayList<Task>} and sets it as a task list
+     * @param taskList The task list that is this new instance wraps around
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Adds a new {@code Task} to the task list
+     * @param task Provided Task that is added to the task list
+     * @return Returns true is addition is successful, else false
+     */
+    public boolean add(Task task) {
+        return taskList.add(task);
+    }
+
+    /**
+     * Removes a {@code Task} from the task list based on the index
+     * @param taskNo Index of the task to be removed
+     * @return The {@code Task} that was removed
+     */
+    public Task remove(int taskNo) {
+        return taskList.remove(taskNo);
+    }
+
+    /**
+     * Get function to get a {@code Task} from the task list
+     * @param taskNo Index of the task to be retrieved
+     * @return The retrieved {@code Task}
+     */
+    public Task get(int taskNo) {
+        return taskList.get(taskNo);
+    }
+
+    /**
+     * Returns the contained task list. Terrible function (will remove soon)
+     * @return The contained task list
+     */
+    public List<Task> getList() { // GET RID OF THIS IN THE FUTURE
```

Comment:
Perhaps you can mark such personal comments with `//TODO` or `//FIXME` and view them in the **TODO tool window** in **IntelliJ**?